app/app.py

```diff
@@ -1,9 +1,9 @@
 from datetime import datetime
 from flask_jwt_extended import jwt_required, get_jwt_identity
 from flask_restful import Resource, reqparse
-from models.models import AppDb
+from models.models import DatabaseModel
 
-AppDao = AppDb('development')
```

Comment:
in create_app you can store the config_name you pass in an environment variable then retrieve it here: 
import os
current_environment = os.environ['CURR_ENV']
*Then pass it in the class call*
AppDao = AppDb(current_environment)

Reply:
yeah it works thats awesome thank you